entry.go

```diff
@@ -40,6 +40,11 @@ func init() {
 	minimumCallerDepth = 1
 }
 
+// Set the global qualified package name.
+func SetSkipPackageNameForCaller(name string) {
```

Comment:
Mere suggestion:
I feel this is more of an `Add`. `Set` sounds like setting _a_ value.
For the user, `SkipPackageFromStackTrace` might make more intuitive sense than the "Caller" internal.

```suggestion
func AddSkipPackageFromStackTrace(name string) {
```